lib/postinstall-win32.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
+
+const { json } = require('mrm-core');
+
+const defaultsName = 'defaults';
+const isWin = process.platform === 'win32';
+const localDir = process
+  .cwd()
+  .split('node_modules')
+  .shift();
+const pkg = json(path.resolve(localDir, 'package.json'));
+const scriptPath = ['scripts', defaultsName];
+
+if (isWin && !pkg.get(scriptPath)) {
+  pkg.set(scriptPath, 'webpack-defaults').save();
+}
```

Comment:
Why only for win32?

Reply:
Because it only does not work on win10, so I guess it works for all other systems, if nobody else is complaining. What do you want me to do?

Reply:
> Because it only does not work on win10

Can you clarify? What is error?

Reply:
Ok, the issue mentioned from above is old, I currently get no errors, but the following is generated:

```
{
  "name": "prefetch-test",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "devDependencies": {
    "@webpack-contrib/defaults": "^6.3.0"
  }
}
```

defaults command is missing under scripts

Reply:
Looks like you want to modify existing `package.json` without run `default` command, right?

Reply:
Ok, I don't what the plugin should do by design.

I assumed that during package installation only the default command is created under scripts and the user executes it manually: `Yarn defaults`.

Yes, the fix just adds the command to `package.json` without executing it.
Should it be executed right away?

Reply:
No, it should create `default` command + for all platforms

Reply:
I don't know how the original `postinstall.js` is currently triggered, because on `win10 ` it is not triggered and does therefore not create the `default `command in `package.json.`

If you want it to be executable on all platforms, then it should be enough, to only add the `postinstall ` hook in `package.json `which in turn executes the exisiting `postinstall.js`.

Reply:
@evilebottnawi I have changed the PR.

The only change compared to the original fork is, that now the existing `post_install.js` is executed during the `postinstall` hook in `package.json`.

This should work on all machines, at least it works on win10, perhaps you can do a test on your machine.

Reply:
On win10 now the installation creates the default command under scripts, but execution still suffers from errors:

**1.)**
![image](https://user-images.githubusercontent.com/966368/84123539-42935b80-aa3a-11ea-96bd-8feb0302db75.png)

**2.)**
![image](https://user-images.githubusercontent.com/966368/84123584-50e17780-aa3a-11ea-9e1d-d71d65633925.png)

**3.)**
![image](https://user-images.githubusercontent.com/966368/84123615-5c34a300-aa3a-11ea-9e3f-ac7807dfa3ca.png)


Everything else is setup fine! Should this PR fix the other errors too? I would rather open another issue and submit another PR.

Reply:
Especially number 2:) should not throw an error or create the `src` folder by itself.

Reply:
For 1.)

```
13 verbose lifecycle prefetch-test@1.0.0~prebuild: CWD: D:\temp\prefetch-test
14 silly lifecycle prefetch-test@1.0.0~prebuild: Args: [ '/d /s /c', 'npm run clean' ]
15 silly lifecycle prefetch-test@1.0.0~prebuild: Returned: code: 1  signal: null
16 info lifecycle prefetch-test@1.0.0~prebuild: Failed to exec prebuild script
17 verbose stack Error: prefetch-test@1.0.0 prebuild: `npm run clean`
17 verbose stack Exit status 1
17 verbose stack     at EventEmitter.<anonymous> (C:\Users\Admin\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\index.js:283:16)
17 verbose stack     at EventEmitter.emit (events.js:315:20)
17 verbose stack     at ChildProcess.<anonymous> (C:\Users\Admin\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
17 verbose stack     at ChildProcess.emit (events.js:315:20)
17 verbose stack     at maybeClose (internal/child_process.js:1021:16)
17 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
18 verbose pkgid prefetch-test@1.0.0
```

Reply:
@evilebottnawi 
Don't read all comments before this one. The PR is good to go. 
But it would be good if you could check it on your maching, I guess u dont use windows..?

The `del-cli` error during `yarn defaults` execution is really weird and it seems not to be related to the plugins implementation but rather somewhere deep below.

Please review.